src/dra/5D6C4.c

```diff
@@ -1126,6 +1149,37 @@ void InitStatsAndGear(bool DeathTakeItems) {
                     AddToInventory(0x52, 0); // Javelin
                     AddToInventory(0x49, 0); // Pentagram
                 }
+#elif defined(VERSION_HD)
+                g_Status.timerHours = 0;
+                g_Status.timerMinutes = 0;
+                g_Status.timerSeconds = 0;
+                g_Status.timerFrames = 0;
+                g_Status.subWeapon = 6;
+                if (rand() & 3) {
+                    g_Status.subWeapon = (rand() % 9) + 1;
+                }
+                do {
+                loop_check_equip_id_1:
+                    equipId = rand() % 169;
+                    if (equipId == 216) {
+                        goto loop_check_equip_id_1;
+                    }
+                } while (D_800A4B04[equipId].itemCategory > 4);
+                g_Status.equipment[0] = equipId;
```

Comment:
It apparently gives a random item for each hand? Makes me curious what the demos are like in that version.